diff --git a/characters/rigs/crayola_default_rig.tscn b/characters/rigs/crayola_default_rig.tscn
index 5eb89eb3a362fc720abd17860864b29fe1d41184..667cc10c777aaeec40ddd28c2925258b31804a96 100644
--- a/characters/rigs/crayola_default_rig.tscn
+++ b/characters/rigs/crayola_default_rig.tscn
@@ -25,50 +25,51 @@ length = 0.3
 [sub_resource type="Animation" id="Animation_oneshot_heavy_attack"]
 resource_name = "oneshot_heavy_attack"
 length = 0.45
 
 
 [sub_resource type="Animation" id="Animation_arm_hip_rest"]
 resource_name = "arm_hip_rest"
 length = 1.0
 loop_mode = 1
 
 [sub_resource type="Animation" id="Animation_arm_wand_ready"]
 resource_name = "arm_wand_ready"
 length = 1.0
 loop_mode = 1
 [sub_resource type="AnimationLibrary" id="AnimationLibrary_1"]
 _data = {
 &"arm_hip_rest": SubResource("Animation_arm_hip_rest"),
 &"arm_wand_ready": SubResource("Animation_arm_wand_ready"),
 &"idle": SubResource("Animation_idle"),
 &"oneshot_heavy_attack": SubResource("Animation_oneshot_heavy_attack"),
 &"oneshot_light_attack": SubResource("Animation_oneshot_light_attack")
 }
 
 [node name="CrayolaRig" type="Node2D" unique_id=87643186]
 script = ExtResource("1_rig")
+aim_vertical_compensation = 14.0
 
 [node name="StomachPivot" type="Node2D" parent="." unique_id=369431116]
 position = Vector2(0, -25)
 
 [node name="Stomach" type="Sprite2D" parent="StomachPivot" unique_id=267454639]
 z_index = 4
 position = Vector2(-5, -16)
 texture = ExtResource("2_stomach")
 centered = false
 
 [node name="ChestPivot" type="Node2D" parent="StomachPivot" unique_id=596155801]
 
 [node name="Chest" type="Sprite2D" parent="StomachPivot/ChestPivot" unique_id=1531807932]
 z_index = 6
 position = Vector2(-5, -16)
 texture = ExtResource("3_chest")
 centered = false
 
 [node name="NeckPivot" type="Node2D" parent="StomachPivot/ChestPivot" unique_id=543778232]
 position = Vector2(-1, -8)
 
 [node name="Neck" type="Sprite2D" parent="StomachPivot/ChestPivot/NeckPivot" unique_id=407874103]
 z_index = 5
 position = Vector2(-4, -8)
 texture = ExtResource("4_neck")
